### DISCUSSION TITLE
MFXFontIcon descriptions

### DISCUSSION THREAD

**Discussion by mspiropali** [palexdev/MaterialFX#229] [Archived]
Hello,

I am new to MaterialFX and I am trying to change the demo stepper icons. In the MFXIconWrapper I have to change the description field from mfx-lock to something else, but I cannot find the list with the available descriptions. In the wiki there is no mention from what I checked. Can you please let me know where to find the supported icon descriptions?

**Comment by palexdev** [comment]
1) In the demo app there's a dedicated section that shows you the icon and its description
2) In the source code, the FontResources.java class

**Reply by TanvirSiddik** [reply]
I've stuck for like 2days. still couldn't figure out how to add a icon to button. tried fontawesome, ikonli not compatible with javafx 23.0.1
help me! please!. 
with MFXFontIcon everything goes fine untill i tried to add the description to the but via fxml i get a long list of error, that i don't even know how to read.

its for my school project help!

**Reply by palexdev** [reply]
> I've stuck for like 2days. still couldn't figure out how to add a icon to button. tried fontawesome, ikonli not compatible with javafx 23.0.1 help me! please!. with MFXFontIcon everything goes fine untill i tried to add the description to the but via fxml i get a long list of error, that i don't even know how to read.
> 
> its for my school project help!

Can you show me the code and errors?
Also what versions of MFX libraries are you using?
